api/proto/config/managed_delivery.proto

```diff
@@ -0,0 +1,113 @@
+syntax = "proto3";
+package proto.config;
+
+option go_package = "github.com/spinnaker/kleat/api/client/config";
+
+import "google/protobuf/wrappers.proto";
+import "storage/sql.proto";
+
+message ManagedDelivery {
+  // Whether ManagedDelivery is enabled.
+    google.protobuf.BoolValue enabled = 1;
+    // Service discovery done through Eureka
+    Eureka eureka = 2;
+    // Keel specific configuration
+    KeelConfig keel = 3;
+
+}
+
+message Keel {
+    // Keel specific configuration
+    KeelConfig keel = 1;
+
+    // Service discovery done through Eureka
+    Eureka eureka = 2;
+
+    // Datastore used for keeping Keel data
+    storage.SQL sql = 3;
+}
+
+message Eureka {
+    // Whether this persistent store is enabled.
+    google.protobuf.BoolValue enabled = 1;
+}
+
+message KeelConfig {
+    // Managed Delivery plugins to be enabled
+    Plugins plugins = 1;
+    // duration string format (eg. 10s)
+    ResourceCheck resourceCheck = 2;
+    // in ISO 8601 format
+    ArtifactRefresh artifactRefresh = 3 [json_name = "artifact-refresh"];
```

Comment:
Does `keel` require kebab case, or will it also properly read snake case?

The reason I ask is that all the other services take config as snake case; this will likely get confusing if we start sharing any of these protos between services.  The other services are on Spring Boot 2 and all still are configured by snake case; I think the same should work for `keel`.  If we do want to generate output in kebab case it probably makes sense to do this for all the services at once.

Reply:
It should handle snake case just fine, since this is Spring Boot and there's nothing specific we're doing in keel. We were just following the best practice from the Spring docs. 🤷‍♂️

Reply:
great thanks for chipping in. Will make it snake case then.